### PR TITLE
Client tests

### PR DESCRIPTION
**Pull request by rabstejnek**
Simple type testing for all client functions to ensure there are no internal errors or overt errors.

### CODE REVIEW

**Comment by rabstejnek** [comment]
@shapiromatron All of the tests are working except the import hero test, which I'm assuming is due to HERO problems. I will keep on running it locally until I can generate a cassette, and then I will push for PR review.

File: tests/data/fixtures/db.yaml [hunk 1]
```diff
@@ -451,6 +451,34 @@
...
+    enable_bmd: true
+    enable_summary_text: true
+    conflicts_of_interest: ''
+    funding_source: ''
+    noel_name: 0
+    rob_name: 0
+    created: 2020-04-10 17:45:39.555285+00:00
+    last_updated: 2020-04-10 17:45:39.555285+00:00
+    project_manager:
+    - - pm@pm.com
```
[the remark this answers is not in the document]
Comment (rabstejnek) [reply tests/data/fixtures/db.yaml:479]: I created this assessment using the GUI and then exported the database, so it should be correct syntax.

[the remark this answers is not in the document]
Comment (shapiromatron) [reply tests/data/fixtures/db.yaml:479]: guess it works then, thanks!

File: tests/client/test_client.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
+import pandas as pd
+import pytest
+from django.core.cache import cache
+from django.test import LiveServerTestCase, TestCase
+
+from hawc_client import BaseClient, HawcClient
+
+
+"""
```
Comment (shapiromatron) [inline tests/client/test_client.py:9]: hey sorry, one super-nit request. can this comment please be moved to be the class-level docstring, and can you make sure the line-width is <100, eg., 

```python
@pytest.mark.usefixtures("set_db_keys")
class TestClient(LiveServerTestCase, TestCase):
    """
    We use a single class that inherits from both LiveServerTestCase and TestCase 
    in order to supersede properties of LiveServerTestCase that cause the database to be flushed 
    after every test, while still being able to utilize a live server for HTTP requests.

    Further reading: https://code.djangoproject.com/ticket/23640#comment:3
    """
```

After that feel free to merge!

**Pull request merged**